cmd/wallet-sdk-gomobile/docs/usage.md

```diff
@@ -789,9 +789,17 @@ val credentials = interaction.requestCredentialWithPreAuth(didDocument.assertion
 
 val issuerURI = interaction.issuerURI() // Optional (but useful)
 
-interaction.requireAcknowledgment() // return true if the issuer requires acknowledgment that credentials are accepted or rejected by the user.
-interaction.acknowledgeSuccess() // acknowledgment issuer that the user accepted credentials.
-interaction.acknowledgeReject() // acknowledgment issuer that the user rejected credentials.
+// Issuance Acknowledgment
+interaction.requireAcknowledgment() // returns true, if the issuer requires acknowledgment that credentials are accepted or rejected by the user.
+interaction.Acknowledgment() // get the Acknowledgment object, if issuer requires ack.
```

Comment:
Acknowledgment and Serialize functions should starts lowercase

Reply:
Updated